pages/linux/pactree.md

```diff
@@ -0,0 +1,24 @@
+# pactree
+
+> Package dependency tree viewer for pacman.
+> More information: <https://man.archlinux.org/man/community/pacman-contrib/pactree.8.en>.
+
+- View the dependency tree of a package:
```

Comment:
We describe the command, the command `print` or `display`, the user view something.
```suggestion
- Print the dependency tree of a specific package:
```